lib/posix/options/clock.c

```diff
@@ -75,6 +93,12 @@ int clock_gettime(clockid_t clock_id, struct timespec *ts)
 		(void)__posix_clock_get_base(clock_id, &base);
 		break;
 
+#ifdef CONFIG_POSIX_THREAD_CPUTIME
+	case CLOCK_THREAD_CPUTIME_ID:
+		(void)__posix_clock_get_base(clock_id, &base);
+		break;
```

Comment:
In the case of execution cycles, I think we don't want to add anything on top of `base`.


```suggestion
   *ts = base;

		return 0;
```

Reply:
I agree, it seems that we only need to get the execution cycle, and no other content is needed.